data_dir/cdc_profile.yaml

```diff
@@ -0,0 +1,75 @@
+keyspace: cdc_test
+
+keyspace_definition: |
+
+  CREATE KEYSPACE cdc_test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '3'}  AND durable_writes = true;
+
+table: test_table
+
+table_definition: |
+
+  CREATE TABLE cdc_test.test_table (
+    primaryrepositoryid text PRIMARY KEY,
+    apiinitiatedtime timestamp,
+    clientappname text,
+    eoid text,
+    eventtime timestamp,
+    fid text,
+    import int,
+    intendedmarket text,
+    intendedroute1 int,
+    intendedroute2 text,
+    messagehash text,
+    messagetype text,
+    mid text,
+    pbrand text,
+    pcn text,
+    potherid text,
+    pothertype text,
+    processtype int,
+    ptype int,
+    pweight decimal,
+    rejectionerrorcode list<text>,
+    rejectionerrordescription list<text>,
+    rejectionerrorinternalid list<text>,
+    reqquantity int,
+    responsetext text,
+    sourcemessageid text,
+    testflag int,
+    tpid text,
+    tppn text,
+    upuis list<text>
+  ) WITH bloom_filter_fp_chance = 0.01
+    AND cdc = {'enabled': true}
+    AND caching = {'keys': 'ALL', 'rows_per_partition': 'ALL'}
+    AND comment = 'Request for unit level UIs'
+    AND compaction = {'class': 'SizeTieredCompactionStrategy'}
+    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
+    AND crc_check_chance = 1.0
+    AND dclocal_read_repair_chance = 0.1
+    AND default_time_to_live = 0
+    AND gc_grace_seconds = 864000
+    AND max_index_interval = 2048
+    AND memtable_flush_period_in_ms = 0
+    AND min_index_interval = 128
+    AND read_repair_chance = 0.0
+    AND speculative_retry = '99.0PERCENTILE';
+
+columnspec:
+  - name: primaryrepositoryid
+    population: exp(1..10000000)
+
+insert:
+  partitions: fixed(1)
+  batchtype: UNLOGGED
+
+queries:
+  read1:
+    cql: select * from cdc_test.test_table where primaryrepositoryid = ?
+    fields: samerow
+  update_fid:
+    cql: update cdc_test.test_table set fid = ? where primaryrepositoryid = ?
+    fields: samerow
+  update_import:
+    cql: update cdc_test.test_table set import = ? where primaryrepositoryid = ?
+    fields: samerow
```

Comment:
Will be good to have the delete query too

Reply:
The delete will cause read on same table fails. Maybe we can use another table, and only write and delete items from it?

Reply:
Yes, this could be good enhancement